textdb/textdb-web/src/main/java/edu/uci/ics/textdb/web/TextdbWebApplication.java

```diff
@@ -52,7 +71,30 @@ public void run(TextdbWebConfiguration textdbWebConfiguration, Environment envir
         cors.addMappingForUrlPatterns(EnumSet.allOf(DispatcherType.class), true, "/*");
     }
 
+    private static void loadStanfordNLP() throws TextDBException{
+
+        // Creating a simple logical plan with a Stanford NLP Extractor operator
+        LogicalPlan logicalPlan = new LogicalPlan();
+        logicalPlan.addOperator(KEYWORD_SOURCE_BEAN.getOperatorID(), KEYWORD_SOURCE_BEAN.getOperatorType(),
+                KEYWORD_SOURCE_BEAN.getOperatorProperties());
+        logicalPlan.addOperator(NLP_EXTRACTOR_BEAN.getOperatorID(), NLP_EXTRACTOR_BEAN.getOperatorType(),
+                NLP_EXTRACTOR_BEAN.getOperatorProperties());
+        logicalPlan.addOperator(TUPLE_STREAM_SINK_BEAN.getOperatorID(), TUPLE_STREAM_SINK_BEAN.getOperatorType(),
+                TUPLE_STREAM_SINK_BEAN.getOperatorProperties());
+        logicalPlan.addLink(KEYWORD_SOURCE_BEAN.getOperatorID(), NLP_EXTRACTOR_BEAN.getOperatorID());
+        logicalPlan.addLink(NLP_EXTRACTOR_BEAN.getOperatorID(), TUPLE_STREAM_SINK_BEAN.getOperatorID());
+
+        // Triggering the execution of the above query plan
+        Plan plan = logicalPlan.buildQueryPlan();
+        TupleStreamSink sink = (TupleStreamSink) plan.getRoot();
+        sink.open();
+        List<ITuple> results = sink.collectAllTuples();
+        sink.close();
+    }
+
     public static void main(String args[]) throws Exception {
         new TextdbWebApplication().run(args);
+
+        loadStanfordNLP();
```

Comment:
The behavior was just to create an instance of the `TextdbWebApplication` class and call the `run()` function of the class. Which can be found on line 96 - ` new TextdbWebApplication().run(args);`